template_feed/Microsoft.DotNet.Common.ProjectTemplates.2.1/content/ConsoleApplication-FSharp/Program.fs

```diff
@@ -1,8 +1,13 @@
-﻿// Learn more about F# at http://fsharp.org
+// Learn more about F# at http://fsharp.org
 
 open System
 
+// Define a function to print a message
```

Comment:
this does not do what the comment says

Also perhaps consider a switch that breaks the option to have such a function out into a template option ? (that's obviously a slippery slope into having a separate Tutorial template of course, and the fact the VS UI is pretty useless for templates doesnt help)

Reply:
Currently the template engine doesn't support this (or rather, it does just by having fully-fledge templates for each switch). I'd rather not go that route if possible.

Reply:
the engine supports it (i.e. the stuff where you define a conditional compilation symbol, use either #ifdefs or //#ifdefs and the engine does preprocessing)- the VS UI doesn't

but yes, that is likely a step too far - the text is better now, thanks!